kubernetes/cmsweb/services/newdqmgui.yaml

```diff
@@ -14,6 +14,7 @@ spec:
     - port: 18889
       targetPort: 18889
       name: newdqmgui-mon
+  type: LoadBalancer
```

Comment:
@micsucmed 

Please remove this LoadBalancer. We use ClusterIP for all backend services.